packages/core/helper-plugin/src/components/Link.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+import { Link as DSLink, LinkProps as DSLinkProps } from '@strapi/design-system/v2';
+import { NavLink } from 'react-router-dom';
+
+const Link = (props: DSLinkProps) => <DSLink {...props} as={NavLink} />;
```

Comment:
This is used in a lot of places but I'm wondering if we should deprecate it and just use the DS Link directly?  Maybe this component is useful so we don't have to `as={NavLink}` everywhere? @joshuaellis what do you think?

Reply:
We should deprecate it yes! :)

Reply:
Cool! @dallasclark could you add @deprecated with a message to use the Design System link instead? Then we can approve and merge